src/remove_merge_checks.py

```diff
@@ -16,6 +19,9 @@ def remove_merge_checks(pull):
     remove_label(pull=pull, label=LABEL_VERIFIED)
     remove_label(pull=pull, label=LABEL_APPROVE)
     remove_label(pull=pull, label=READY_FOR_MERGE)
+    for label in SEMVER_LABELS_SET:
```

Comment:
Didn't we agree not to remove the labels?

Reply:
Any new code pushed may change the scope of the semver label. Previously assigned semver labels must be considered invalid after any push.